### ISSUE TITLE
Playing this decklist sometimes causes an exception in the client

### ISSUE THREAD

**Issue by doctorpangloss** [hiddenswitch/Spellsource-Server#1339]
Decklist:

```
### Spirit Power
# Class: JADE
# Format: Spellsource
#
# 2x (1) Bromediad Pup
# 1x (1) Doodles
# 2x (1) Palm Strike
# 2x (2) Chi Wave
# 2x (2) Dreamwing Scout
# 2x (2) Floating Crystal
# 1x (2) Hoffis the Dunewalker
# 2x (2) Ninja Aspirants
# 2x (2) Strengthen Core
# 2x (3) Chromatic Vohlok
# 1x (3) Klive Icetooth
# 2x (3) Windswept Strike
# 2x (4) Lucid Landscape
# 2x (5) Chi Restoration
# 2x (5) Lunging Jab
# 1x (5) The Imperial Soul
# 2x (6) Bookkeeper
```

User: @Angry2

**Comment by doctorpangloss** [comment]
[www.playspellsource.com-1565552130457.log](https://github.com/hiddenswitch/Spellsource-Server/files/3490380/www.playspellsource.com-1565552130457.log)

out-of-memory logs

**Comment by doctorpangloss** [comment]
fixed since 3.0.0
